day-2/advice/src/App.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+var quotes = ['Kick ass today', 
+              'Act like you know what you\'re doing',
+              'Watch out for the fucking scope creep',
+              'Make sure there\'s money in the bank',
+              'Get that git flow', 
+              ]
+
+var _this;
+
+class App extends Component {
+    constructor() {
+      super();
+      _this = this;
+      this.state = {
+        index:0 
```

Comment:
index is fine in this application because it is so simple. But generally it would be best to name it `quoteIndex` or make a quote component with an index state variable